setup.sh

```diff
@@ -2,7 +2,7 @@
 
 
 
-pip3 install jax jaxlib pandas tqdm feedparser pyarrow
+pip3 install jax jaxlib pandas tqdm feedparser pyarrow beautifulsoup4 huggingface_hub[cli]
```

Comment:
add missing dependencies. Note that I'm not super sure if it should be `huggingface_hub[cli]` or `huggingface_hub\[cli\]`